internal/schema/0.12/variable_block.go

```diff
@@ -34,7 +34,7 @@ func variableBlockSchema(v *version.Version) *schema.BlockSchema {
 				Description:            lang.PlainText("Variable Name"),
 			},
 		},
-		Description: lang.Markdown("Input variable allowing users to customizate aspects of the configuration when used directly " +
+		Description: lang.Markdown("Input variable allowing users to customize aspects of the configuration when used directly " +
```

Comment:
While this is a needed typo fix, this isn't stack related so it makes it look like this PR handles terraform stuff

Reply:
oh, yeah, will move into a separate commit 👍

Reply:
https://github.com/hashicorp/terraform-schema/pull/362